pom.xml

```diff
@@ -167,6 +167,31 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
```

Comment:
Great! Maybe we can add some instructions to the README file? Do we need to add some rules?

Reply:
Sure, I can do that. I was thinking for now it can be just for the files changed

Reply:
Added a note in `CONTRIBUTING.md`!